webapp/models.py

```diff
@@ -53,6 +54,9 @@ class CVE(db.Model):
     published = Column(DateTime)
     description = Column(String)
     ubuntu_description = Column(String)
+    updated_at = Column(
+        DateTime(timezone=True), server_default=func.now(), onupdate=func.now()
```

Comment:
Can I check something? Initially I thought that `published` was automatically set by us when a CVE record was created, so I was going to say we should set `updated_at` in the same way as `published`.

But looking at the code I think maybe I've misunderstood this. Is `published` in fact set by the client only, and therefore we have no opinion on what it is and its relation to `updated`? In which case, what you've done here is 100% right.